test/metabase/models/setting_test.clj

```diff
@@ -199,30 +199,27 @@
               #"Cannot initialize setting before the db is set up"
               (setting/get :test-setting-custom-init)))))))
 
-(def ^:private fancy-type
-  {:type :json
-   :setter :none
-   :init str
-   :bloop? true})
-
-(deftest inject-base-test
-  (testing "The interaction between different :type values and the remaining options"
-    (let [expand            #'setting/inject-base
-          explicit-settings {:bloop? false :blep? 'obviously}
-          with-basic-type   (assoc explicit-settings :type :json)
-          with-map          (assoc explicit-settings :base fancy-type)
-          with-var          (assoc explicit-settings :base (symbol ::fancy-type))]
-      (testing "There's no magic when using regular keyword types"
-        (is (= with-basic-type (expand with-basic-type))))
-      (let [expected {:type   :json
-                      :setter :none
-                      :init   str
-                      :bloop? false
-                      :blep?  'obviously}]
-        (testing "You can use explicit maps for the mixin behaviour"
-          (is (= expected (expand with-map))))
-        (testing "You can use var references for the mixing behaviour"
-          (is (= expected (expand with-var))))))))
+(def base-options
```

Comment:
```suggestion
(def ^:private base-options
```

It's always nice to make stuff that's only used in this specific test namespace `^:private`, so we can change it with worrying about it affecting stuff in other test namespaces, and so Kondo can warn us about it being unused if we change the test that was using it so it doesn't become dead code

Reply:
Those are great reasons, thanks for pointing them out. I guess I've taken those assurances for granted in the languages I've been working in before, where tests are written in private modules or classes.